src/wakatimeScript.ts

```diff
@@ -249,7 +257,17 @@ const init = async () => {
 
   const { hostname } = document.location;
   const canvaProject = document.getElementsByClassName('rF765A');
+  const figmaProject = document.getElementsByClassName('gpu-view-content');
 
+  if (hostname === 'www.figma.com' && figmaProject.length > 0) {
```

Comment:
Looks like we'v solve the same issue #184
I think it will be better to do not follow "if elseif elseif" pattern because it's quite difficult to support in future for multiple domains for example.
@jvelezpo what do you think about it, are you agree?
Probably, you are not going to support anything else except Canva and Figma, so there are no concerns here.